Directory.Build.props

```diff
@@ -0,0 +1,18 @@
+<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
+
+  <PropertyGroup>
+    <Copyright>$(CopyrightNetFoundation)</Copyright>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+    <DebugType>embedded</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+    <LangVersion>Latest</LangVersion>
+    <IsPackable>true</IsPackable>
```

Comment:
Not sure if the plan is currently to move all experiments from corefxlab to runtimelab. If it is, this property defaults to false in corefxlab. Otherwise, ignore this comment!

Reply:
I set this to true here and then test set it to false. In corefxlab we have more projects and we wanted to intentionally enable it because not al projects are packable, here I expect the majority to be, so I think it makes sense to default to true.